src/karma-webpack.js

```diff
@@ -17,6 +17,15 @@ var isBlocked = false
 
 const normalize = (file) => file.replace(/\\/g, '/')
 
+var getJsOutput = (outputPathArray) => {
+  for (var _i = 0; _i < outputPathArray.length; _i++) {
+    if (outputPathArray[_i].indexOf(".js") != -1) {
```

Comment:
How des this work? Could you give a five examples please? What  happens if e.g `file.js.map` takes precedence?

Reply:
```diff
- !=
+ !==
```

Reply:
> How des this work? Could you give a five examples please? What happens if e.g file.js.map 
> takes precedence?

You are right. I did not consider this case actually? I have in my webpack config 3 entries. like
entry: {
    app: `${scriptsPath}/ibe/${IBEClient}/app.ts`,
    vendor: `${scriptsPath}/vendor.browser.ts`,
    polyfills: `${scriptsPath}/polyfills.browser.ts`,
}
So, I move the entries to Karma config to get the same output form webpack. and remove it form webpack config. mostly I didn't see this case because is the .js output is always before file.js.map output in all my cases. The problem I faced is if the webpack shanks has .css.map as first output.

In any case i will add .js.map to if condition which should solve this case also :)